$NetBSD: patch-ikiwiki.in,v 1.1 2024/02/05 06:13:44 schmonz Exp $

Apply upstream patch 3b24262 to fix warnings with newer Getopt::Long.

--- ikiwiki.in.orig	2024-02-05 06:11:22.083293105 +0000
+++ ikiwiki.in
@@ -31,8 +31,8 @@ sub getconfig () {
 		Getopt::Long::Configure('pass_through');
 		GetOptions(
 			"setup|s=s" => \&setup,
-			"dumpsetup|s=s" => \$config{dumpsetup},
-			"changesetup|s=s" => sub {
+			"dumpsetup=s" => \$config{dumpsetup},
+			"changesetup=s" => sub {
 				$config{changesetup}=$_[1];
 				$config{genwrappers}=1;
 				$config{refresh}=1;
